src/components/AudioResultsList.vue

```diff
@@ -7,8 +7,8 @@
     </div>
     <div class="px-6 pt-6">
       <AudioTrack
-        v-for="audio in results.items"
-        :key="audio.id"
+        v-for="(audio, index) in results.items"
+        :key="index"
```

Comment:
What is this change for?

Reply:
This is just to make the code for audio same as that for images. The output is the same because the `index` is the ID itself.